### ISSUE TITLE
Mod Support Not Working?

### ISSUE THREAD

**Issue by uhmbye** [lazerzes/Anvil-Enchantments#22]
I sent you a DM on CurseForge but I figured this is probably the better place to report this.  And I may just be using the mod wrong, and if so could you please elaborate how I do this. But I have multiple custom anvil recipes that use items from other mods, however, they don't work. For example, one of my recipes [look like this](https://gyazo.com/0ddcce522ce8a810d8478fe572d6855a), which uses a modded item. I can use vanilla items, even with custom enchantments added by other mods, but any scripts that uses a domain from another mod doesn't work.

An example of one of my scripts:

```
{
    "recipeName": "Rust",
    "anvilLeft": {
        "registry": {
            "domain": "minecraft",
            "path": "book"
        },
        "count": 1,
        "meta": 0
    },
    "anvilRight": {
        "registry": {
            "domain": "quark",
            "path": "quark:iron_plate"
        },
        "count": 1,
        "meta": 1
    },
    "anvilOutput": {
        "registry": {
            "domain": "minecraft",
            "path": "enchanted_book"
        },
        "nbt": {
            "data": "{StoredEnchantments:[{lvl:1s,id:26s}]}"
        },
        "count": 1,
        "meta": 0
    },
    "anvilLevelCost": 26,
    "showRecipe": true
}
```

**Comment by sensdoggo** [comment]
Sorry for the late response.

I believe the issue here is actually rather easy to solve.

anvilRight's registry is broken into 
    domain ---> mod id
    path --> item name

So instead of ...

```"anvilRight": {
        "registry": {
            "domain": "quark",
            "path": "quark:iron_plate"
        },
        "count": 1,
        "meta": 1
    },
```

... you should have.

```"anvilRight": {
        "registry": {
            "domain": "quark",
            "path": "iron_plate"
        },
        "count": 1,
        "meta": 1
    },
```

Sorry for any confusion this may have caused. Please confirm that this change does indeed work for you as well. Thank you!

**Comment by uhmbye** [comment]
Sadly this does not work either. Originally I used the format of domain: "modname" and path: "item_name" but the results were still nothing. I've tried it multiple ways including the way in my original post but sadly just doesn't work.

The issue I think is that the mod isn't reading the domain if it's not "minecraft", or something around those lines. Thanks for the reply though.

**Comment by sensdoggo** [comment]
Hmm, I'll take a deeper look into this in the next few days.

Thank you for your patience.

**Comment by sensdoggo** [comment]
This has been fixed in version 2.0.3,

You will want to use the config change I suggested earlier as well.

Please try it out and let me know.

**Comment by sensdoggo** [comment]
Closed for inactivity
